react/ProductHighlights.tsx

```diff
@@ -36,11 +26,7 @@ function createFilterHighlight(filter: Filter) {
   }
 }
 
-const ProductHighlights: FC<ProductHighlightsProps> = ({
-  filter = defaultFilter,
-  type = 'collection',
-  children,
-}) => {
+const ProductHighlights = (props: { filter: Filter; type: HighlightType; children: NonNullable<ReactNode> }) => {
```

Comment:
You should leave the props as they were.

```suggestion
const ProductHighlights: FC<ProductHighlightsProps> = ({
  filter = defaultFilter,
  type = 'collection',
  children,
}) => {
```